cosmwasm/contracts/global-accountant/Cargo.toml

```diff
@@ -37,3 +37,4 @@ anyhow = { version = "1", features = ["backtrace"] }
 cw-multi-test = "0.13.2"
 serde-json-wasm = "0.4"
 wormhole-bindings = { version = "0.1", features = ["fake"] }
+serde_json = "1.0"
```

Comment:
We already have serde-json-wasm in the dev-dependency list.  Can we reuse that?

Reply:
no longer need it after removing need to serialize `Modification `